### PR TITLE
chore(ci): pre-release dependabot

### PR DESCRIPTION
**Pull request by TarikGul**
This PR builds on issue [#380 ](https://github.com/paritytech/substrate-api-sidecar/issues/380).

This PR has changed from creating a automated publishing pipeline to a sole Dependabot script to create PR's every Monday with updated packaging from `@polkadot/..`. 

After some discussion and thought, it would make sense to do each section of automated publishing in seperate PR's and to build out the process piece by piece. 

```

 | Every Monday at XX:XX(am/pm) check for updates in the following 3 packages |
 |        1                   2                       3                       |
 | @polkadot/api => @polkadot/apps-config => @polkadot/util-crypto.           |
 | When the updates exists, create the PR.                                    |
  
```

### CODE REVIEW

**Review by emostov — APPROVED** [review]

**Review by dvdplm — APPROVED** [review]
Nice, it's more succinct than I thought it'd be.

cc @niklasad1 Maybe we can apply the same technique to other places? Nothing specific comes to mind, but good to know it's there. :)

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      # 11 am PST
+      time: "11:00"
+      # PST timezone
+      timezone: "America/Los_Angeles"
+    rebase-strategy: "auto"
+    commit-message:
```
Comment (TriplEight) [inline .github/dependabot.yml:14]:
```suggestion
    open-pull-requests-limit: 20
    commit-message:
```

Comment (TriplEight) [reply .github/dependabot.yml:14]: In case you don't want to be spammed with these PRs, feel free to change the number.

**Review by TriplEight — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (TarikGul) [reply .github/dependabot.yml:14]: Good call, I'll definitely add it in, for now I think 10 will do. I saw that 5 is the default so I think having a little more room for open pull requests would definitely be beneficial. Thanks Denis.

**Pull request merged**